### ISSUE TITLE
Errors are not reported

### ISSUE THREAD

**Issue by grund3g** [laravel/framework#25360]
- Laravel Version: 5.6.28
- PHP Version: 7.2.2

### Description:
I open this issue because it is not clear if the error is only due to laravel-bugsnag or laravel itself.

As described in this issue https://github.com/bugsnag/bugsnag-laravel/issues/311 errors are not or only a fraction of the errors are sent to Bugsnag.

### Steps To Reproduce:
Using laravel valet (installed a fresh copy of laravel 5.6.28)

```
laravel new bugsnagtest
composer require "bugsnag/bugsnag-laravel:^2.0"
```
followed these instructions: https://docs.bugsnag.com/platforms/php/laravel/
added `Bugsnag\BugsnagLaravel\BugsnagServiceProvider::class`, to the providers array in `app/config.php`

added `BUGSNAG_API_KEY=your-api-key-here` to my `.env` with an api-key of an new / empty Bugsnag project

updated as mentioned in the docs my `config/logging.php`

and finally as mentioned in the docs `Bugsnag::notifyException(new RuntimeException("Test error"));`

The error is only sent the first time to bugsnag.

**Comment by sisve** [comment]
I understand that you're desperate to find the cause of the problem, but what makes you think it's a problem in laravel/framework? Is there anything at all that points to it?

Is it even possible for any Laravel developer to reproduce or track down this issue without being bugsnag customers and having a bugsnag api key?

Can we do anything at all with this issue as-is?

**Comment by grund3g** [comment]
I don't know, and since bugsnag hasn't been able to trace the problem so far, it would be interesting if someone with more experience of the laravel core could look at it.

No, without Bugsnag account you won't be able to reconstruct this, but since Bugsnag is extensively used in the whole laravel community, it affects some people (see also issue above, where others have the same problem).

Bugsnag has worked without any problems until now. The problems occurred only since the logging changes in 5.6.

**Comment by browner12** [comment]
I will say that my Bugsnag has been working just fine on v5.6, so this is most likely an issue with your application.  I would close this issue and try asking in a forum for better help.

**Comment by grund3g** [comment]
I also occurs on a fresh laravel installation as described above.

**Comment by it-can** [comment]
@Sladewill on laravel 5.6 it is set in ```config/logging.php```

**Comment by grund3g** [comment]
@Sladewill as mentioned in the docs, this is now done in the `BugsnagServiceProvider` and no longer in the `AppServiceProvider`.

**Comment by Sladewill** [comment]
I believe we are running an older version of bugsnag laravel and haven't got it running through the logger on our 5.6 projects. Which appears to be working.

**Comment by jmarcher** [comment]
@grund3g Are you sure that the error is being sent only one time? Bugsnag collapses all errors of the same kind. You can see how many occurrences an error had, I believe this is your case.

**Comment by grund3g** [comment]
@jmarcher, yes it is sent only one time. Always "1" in the "Events" row in my Bugsnag Inbox

**Comment by grund3g** [comment]
I also checked if the "error request" is sent at all or not to the bugsnag server with a network analyzer, but it seems that the error never leaves my app
